c-api/type.po

```diff
@@ -6,43 +6,54 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2017-04-02 22:11+0200\n"
-"PO-Revision-Date: 2018-02-15 00:33+0100\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2018-09-20 21:33+0200\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 1.8.11\n"
 
 #: ../Doc/c-api/type.rst:6
 msgid "Type Objects"
-msgstr "Objets type"
+msgstr "Type d'objets"
 
 #: ../Doc/c-api/type.rst:13
 msgid "The C structure of the objects used to describe built-in types."
 msgstr ""
+"La structure en C des objets avait pour habitude de décrire les types natifs "
+"en python."
 
 #: ../Doc/c-api/type.rst:18
 msgid ""
 "This is the type object for type objects; it is the same object as :class:"
 "`type` in the Python layer."
 msgstr ""
+"Ceci est le type d'objet pour les autres types d'objets; C'est le même objet "
+"que :class:`type` dans la couche de Python."
 
 #: ../Doc/c-api/type.rst:24
 msgid ""
 "Return true if the object *o* is a type object, including instances of types "
 "derived from the standard type object.  Return false in all other cases."
 msgstr ""
+"Retourne vrai si l'objet *o* est un type d'objet y compris les instances de "
```

Comment:
Pa souci de cohérence dans l'ensemble de la documentation, le glossaire de README.rst indique "renvoyer" pour "to return".